src/adapter/src/catalog.rs

```diff
@@ -2711,17 +2711,23 @@ impl Catalog {
             builtin_table_updates.push(catalog.state.pack_audit_log_update(&event)?);
         }
 
-        // To avoid reading over storage_usage() multiple times, do both the
-        // table updates and most-recent-timestamp calculations on a single
-        // iterator.
-        let storage_usage_events = catalog.storage().await.storage_usage().await?;
         // If no usage retention period is set, we set it to an unreasonably large number
```

Comment:
Can this comment be changed since 0 is not "unreasonably large" (and I got confused and worried for some minutes that we maybe needed MAX instead)?